recognition/include/pcl/recognition/face_detection/face_common.h

```diff
@@ -151,14 +156,15 @@ namespace pcl
             for (std::size_t j = 0; j < 3; j++)
               stream.read (reinterpret_cast<char*> (&covariance_rot_ (i, j)), sizeof(covariance_rot_ (i, j)));
 
+          if (num_of_sub_nodes < 0)
```

Comment:
Remove the throw and just have an empty return instead

Reply:
Alright, but could you elaborate on why?